gulpfile.js

```diff
@@ -116,7 +115,6 @@ gulp.task('build-sass', () => {
       cascade: false,
       grid: true,
     }),
-    mqpacker({ sort: true }),
```

Comment:
is there some other kind of CSS sort we can use instead to get the determinism back?

Reply:
> is there some other kind of CSS sort we can use instead to get the determinism back?

Yeah, there's at least [two options](https://github.com/hail2u/node-css-mqpacker#sort) here. To be honest, I didn't spend much time considering it, since I don't really think this package is worth keeping around, all things considered (it's deprecated, maintenance overhead, unpredictability of build-time reordering, actually increasing the size of the gzipped bundle).

Reply:
No I meant some sort of other part of the build process instead of `mqpacker` -- not options within `mqpacker`

Reply:
> No I meant some sort of other part of the build process instead of `mqpacker` -- not options within `mqpacker`

Ah. I don't think we want the build to manage this. Instead, we should want to rely on manual ordering of our imports (e.g. [settings](https://github.com/18F/identity-style-guide/blob/d2d697bb95bd9677e002d06fea1ac7d62c1b2902/src/scss/styles.scss#L3-L9), [dependencies](https://github.com/18F/identity-style-guide/blob/d2d697bb95bd9677e002d06fea1ac7d62c1b2902/src/scss/styles.scss#L11), [overrides and extras](https://github.com/18F/identity-style-guide/blob/d2d697bb95bd9677e002d06fea1ac7d62c1b2902/src/scss/styles.scss#L13-L30), [similar within USWDS itself](https://github.com/uswds/uswds/blob/b0e7ea3800e5bb5960e0925716cf5363375ee613/src/stylesheets/uswds.scss#L3-L17)).

Or maybe I'm still not understanding 😅   As far as I can tell, there _was_ an expected determinism by how we order the stylesheets, and the expectation was broken due to how mqpacker works to regroup the output styles.

Reply:
Ok! So we did want manual ordering, I think it all makes sense now